src/main/java/com/vmware/vipclient/i18n/messages/api/opt/server/RemoteProductOpt.java

```diff
@@ -52,7 +52,7 @@ public ProductBasedOpt(BaseDTO dto) {
      * @see com.vmware.vipclient.i18n.messages.dao.ILocaleDao#getSupportedLocales()
      * @deprecated Replaced by {@link com.vmware.vipclient.i18n.messages.api.opt.server.RemoteLocaleOpt#getSupportedLanguages(String)()}
      */
-    @Deprecated public JSONArray getSupportedLocalesFromRemoteVIP() {
+    public List<String> getSupportedLocales() {
         JSONArray msgObject = new JSONArray();
         String responseStr = "";
         Map<String, Object> response = VIPCfg.getInstance().getVipService().getHttpRequester().request(V2URL.getSupportedLocaleListURL(
```

Comment:
The plan is to deprecate this V2URL on service side because it is but a subset of V2URL.getSupportedLanguageListURL (language tags only instead of tags + display names). So call LocaleService.getDisplayNames(null) in here, just grab the keySet(), and it will achieve the same result.
Note: See Code suggestion in LocaleService.getDisplayNames() first.